packages/Tag/styles.ts

```diff
@@ -60,6 +76,12 @@ export const Tag = styled.div<StyledTagProps & WuiProps>(
       margin-right: xxs;
     }
 
+    & > svg:only-child,
+    & > svg:not(:only-child) {
```

Comment:
I think I'm wrong but how is this rule different from `& > svg` ? (since it's targeting only-child AND not only-child?)

Reply:
You are not wrong, we could simplify it by `& > svg` 😅 quite simply 🙈